src/java/htsjdk/samtools/Defaults.java

```diff
@@ -19,9 +19,18 @@
     /** Should MD5 files be created when writing out SAM and BAM files?  Default = false. */
     public static final boolean CREATE_MD5;
 
-    /** Should asynchronous I/O be used when writing out SAM and BAM files (one thread per file).  Default = false. */
+    /** Should asynchronous I/O be used when writing out SAM, BAM, FASTQ files (one thread per file).  Default = false. */
+    @Deprecated
     public static final boolean USE_ASYNC_IO;
 
+    /** Should asynchronous I/O be used when writing out SAM and BAM files (one thread per file).  Default = false. */
+    public static final boolean USE_ASYNC_IO_FOR_SAMTOOLS;
+
+    /** Should asynchronous I/O be used when reading and writing out files processed by Tribble, eg VCF (one thread per file).
```

Comment:
It's awkward that the first two are for writing only, and the tribble option is for both reading and writing. Perhaps it would be better to say "Should asynchronous I/O be used where supported in tribble/samtools/etc." in the docs for these options.